syntax = "proto3";

package overlay;

service Authentication {
	rpc AuthenticateWithToken (TokenAuthenticationRequest) returns (TokenAuthenticationReply) {}
}

message TokenAuthenticationRequest {
	bytes token = 1;
}

message TokenAuthenticationReply {

}